.github/workflows/build-test.yml

```diff
@@ -64,5 +64,5 @@
        if: success() || steps.test.conclusion == 'failure'
        uses: actions/upload-artifact@v3
        with:
-         name: my-artifact
+         name: test-results
```

Comment:
Trying to sift through tons of logs for an error is hard, so I added an HTML test report that GH Actions will upload. I forgot to rename it when I added this workflow.